### DISCUSSION TITLE
Using profile mode on a completely separate page

### DISCUSSION THREAD

**Discussion by jeffreyjsung** [adityatelange/hugo-PaperMod#777] [Q&A] [answered]
How do I use profile mode on a completely separate page (not the home page)?

**Comment by adityatelange** [comment, marked as answer]
- Copy https://github.com/adityatelange/hugo-PaperMod/blob/master/layouts/partials/index_profile.html
- Paste it in `/layouts/_default/`
- Add https://github.com/adityatelange/hugo-PaperMod/blob/530044033e8a9d87d5297b936e37cbfb2b5562e3/layouts/_default/archives.html#L1 and https://github.com/adityatelange/hugo-PaperMod/blob/530044033e8a9d87d5297b936e37cbfb2b5562e3/layouts/_default/archives.html#L49 to Top and bottom of `/layouts/_default/index_profile.html`
- Use the below template in separate `/content/page.md`
    ```md
    ---
    layout: index_profile
    ---
    ```
